spec/lib/repo/git_revision_spec.rb

```diff
@@ -12,7 +12,13 @@
 
       # Make sure repo exists, if not make it
       repo = ga_repo.config.get_repo('test_repo_workdir')
+
       if repo.nil?
+        Repository.conf = {
+            REPOSITORY_STORAGE: REPOSITORY_STORAGE,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.